src/ckeditor.js

```diff
@@ -82,6 +85,12 @@ Editor.defaultConfig = {
 	table: {
 		contentToolbar: ["tableColumn", "tableRow", "mergeTableCells"],
 	},
+	math: {
+		engine: 'katex',
+		outputType: 'span',
+		forceOutputType: true,
+		enablePreview: true,
+	},
```

Comment:
are there already ideas which options to implement? 
https://katex.org/docs/options.html

Reply:
No, for now we only have this basic configuration; same to the one used in legacy client regarding outputType and preview. In contrast to legacy we force output to be a span - and not a script (as we strip this). When talking about migrations at some point, this configuration and also KaTex config might be a topic indeed